### PR TITLE
use htmlhelpter in cshtml

### PR DESCRIPTION
**Pull request by SergKanz**
Addressing @Eilon feedback on integration points

### CODE REVIEW

**Comment by abaranch** [comment]
Approved

File: src/Microsoft.ApplicationInsights.AspNet/ApplicationInsightsJavaScript.cs [hunk 1]
```diff
@@ -13,7 +15,7 @@ namespace Microsoft.ApplicationInsights.AspNet.JavaScript
     ///         @Html.Raw(ApplicationInsightsJavaScript.Write())
```
Comment (upendras) [inline src/Microsoft.ApplicationInsights.AspNet/ApplicationInsightsJavaScript.cs]: nit - we ned to update the comment

**Comment by upendras** [comment]
:shipit:

File: src/Microsoft.ApplicationInsights.AspNet/ApplicationInsightsJavaScript.cs [hunk 2]
```diff
@@ -1,19 +1,15 @@
...
-    /// Usage:
-    ///     Inject an instance of the class into your razor code using code:
-    ///         @inject Microsoft.ApplicationInsights.AspNet.JavaScript.IJavaScriptSnippet ApplicationInsightsJavaScript
-    ///     Place this code fragment just before closing "head" tag
-    ///         @Html.Raw(ApplicationInsightsJavaScript.Write())
-    ///     &lt;/head&gt;
     /// </summary>
-    public class ApplicationInsightsJavaScript : IJavaScriptSnippet
+    public class JavaScriptSnippet
     {
```
Comment (olegsych) [inline src/Microsoft.ApplicationInsights.AspNet/ApplicationInsightsJavaScript.cs]: So at this point, `JavaScriptSnippet` is an implementation detail of the `ApplicationInsightsJavaScript` extension method and does not need to be public. Correct? Is that something we care about?

**Pull request merged**